crates/sui-transaction-builder/src/lib.rs

```diff
@@ -260,8 +258,14 @@ impl<Mode: ExecutionMode> TransactionBuilder<Mode> {
         gas_budget: u64,
     ) -> anyhow::Result<TransactionData> {
         let single_move_call = SingleTransactionKind::Call(
-            self.single_move_call(package_object_id, module, function, type_args, call_args)
-                .await?,
+            self.single_move_call::<execution_mode::Normal>(
```

Comment:
I thought we don't want to leak `execution_mode` to transaction builder (https://github.com/MystenLabs/sui/pull/7236/files), is there anything changed?

Reply:
I do not understand why you wouldn't want to leak it to the builder. You would want to be able to specify why set of rules to run, no?